.github/workflows/greetings.yml

```
@@ -21,13 +21,11 @@ jobs:
             ## **Thank you for contributing to the Convert2RHEL project!**
             👋 Hello @${{ github.actor }}, thank you for submitting a PR 🚀!
             Please note that every PR needs to comply with the [Convert2RHEL Guidelines](https://github.com/oamg/convert2rhel/blob/main/CONTRIBUTING.md) and must pass all tests in order to be mergable.
-            If you want to rebuild a package in copr, you can use following commands as a comment:
-            - **/packit copr-build** to submit a public copr build using packit
+            If you want to rebuild a package in [Fedora Copr](https://copr.fedorainfracloud.org/coprs/g/oamg/convert2rhel/), you can use the following command as a comment:
+            - **/packit copr-build** to trigger a copr build using [packit](https://packit.dev/)
 
-            To launch regression testing public members of oamg organization don't need any extra step. The automation kicks that off for you!
+            To execute integration tests, members of [the oamg organization](https://github.com/oamg/) don't need any extra step. The automation kicks that off for you!
 
             If you're an external collaborator (Meaning that you're not part of oamg), ping the `@oamg/convert2rhel-developers` group to assign the testing label to your pr.
```

As I'm thinking about it, maybe we shouldn't be telling them to ping us. We should know what to do. And if somebody creates a PR for us we want the tests to pass so we will add the label even without them pinging us.

So I'd change it to:

```suggestion
            If you're an external collaborator (meaning that you're not part of the oamg organization), wait for somebody from the @oamg/convert2rhel-developers group to add the testing label to the PR to trigger the tests.
```